bld/namelist_files/namelist_definition_ctsm.xml

```diff
@@ -794,9 +799,22 @@ Full pathname to the inventory initialization control file.
 <entry id="use_fates_luh" type="logical" category="physics"
        group="clm_inparm" valid_values="" >
 If TRUE, enable use of land use harmonization (LUH) state and transition data from luh_timeseries file.
+This is enabled by default if fates_harvest_mode is set to use the raw LUH2 harvest data
 (Also, only valid for use_fates = true and is incompatible with transient runs currently.)
 </entry>
 
+<entry id="use_fates_lupft" type="logical" category="physics"
+       group="elm_inparm" valid_values="" value=".false.">
```

Comment:
Change name from elm_inparm (should be clm_inparm).

Reply:
addressed via https://github.com/ESCOMP/CTSM/pull/2507/commits/3ff5f4a533daff344b32c26e015338b0ebf565ed